src/librustc_metadata/encoder.rs

```diff
@@ -635,13 +641,8 @@ impl EncodeContext<'tcx> {
         let data = VariantData {
             ctor_kind: variant.ctor_kind,
             discr: variant.discr,
-            // FIXME(eddyb) deduplicate these with `encode_enum_variant_ctor`.
+            // FIXME(eddyb) deduplicate this with `encode_enum_variant_ctor`.
```

Comment:
I don't understand what "duplicate this with" means. Do you mean "keep this in sync with"?

Reply:
"deduplicate" as in there should be a helper or (even better) this code should exist only in one place.
The comment is old anyway, I think it predates @petrochenkov's revamp of how enum/struct ctors work.